meta-lmp-base/recipes-containers/composeapp/composectl_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "A CLI utility to manage compose apps"
+
+SECTION = "devel"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=2a944942e1496af1886903d274dedb13"
+
+GO_IMPORT = "github.com/foundriesio/composeapp"
+GO_IMPORT_PROTO ?= "https"
+SRC_URI = "git://${GO_IMPORT};protocol=${GO_IMPORT_PROTO};branch=main"
+SRCREV = "806593af2ccc5d9cca7233e0bbf12ca778fe3b1a"
```

Comment:
while in WIP you can use `SRCREV = "${AUTOREV}"` and bitbake will pick-up the HEAD of the branch

Reply:
I just thought I would not need to update the commit :).